src/app_service/service/community/service.nim

```diff
@@ -804,7 +804,7 @@ QtObject:
               self.joinedCommunities[communityId].chats[prev_chat_idx].position = chat.position
               chatDetails.updateMissingFields(self.joinedCommunities[communityId].chats[prev_chat_idx])
               self.chatService.updateOrAddChat(chatDetails) # we have to update chats stored in the chat service.
-              self.events.emit(SIGNAL_COMMUNITY_CHANNEL_REORDERED, CommunityChatOrderArgs(communityId: updatedCommunity.id, chatId: fullChatId, categoryId: chat.categoryId, position: chat.position))
+              self.events.emit(SIGNAL_COMMUNITY_CHANNEL_REORDERED, CommunityChatOrderArgs(communityId: updatedCommunity.id, chatId: chat.id, categoryId: chat.categoryId, position: chat.position))
```

Comment:
`fullChatId` doesn't work with `self.chatService.getChatById`, plain `chat.Id` is expected as well. When `fullChatId` is used, chat details is not found and chatService is not updated. Error is msg is emitted to the output as well.